packages/services/api/src/modules/shared/providers/id-translator.ts

```diff
@@ -156,6 +156,58 @@ export class IdTranslator {
 
     return selector;
   }
+
+  /** Resolve a GraphQLSchema.OrganizationReferenceInput */
+  async resolveOrganizationReference(args: {
+    reference: GraphQLSchema.OrganizationReferenceInput;
+    onError(): never;
+  }): Promise<{ organizationId: string }> {
+    let selector: {
+      organizationId: string;
+    };
+
+    if (args.reference.bySelector) {
+      const organizationId = await this.translateOrganizationId(args.reference.bySelector).catch(
+        error => {
+          this.logger.debug(error);
```

Comment:
Not as part of this PR, but I think we should standardize our logs to include a unique code so that we can easily track log lines back to their source.
I think as they are now, if i saw these lines in a log that I wouldn't know what went wrong.